app/views/categories/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<section class="categories--container">
+  <article class='form--header'>
+    <h2>Categories</h2>
+    <%= link_to 'LOG OUT', destroy_user_session_path, data: {turbo_method: :delete} %>
+  </article>
+  <article class="catergories--list--container">
+    <ul class="categories--list">
+      <% @categories.each do |category| %>
+        <li class="categories--item">
+        <div class="categories--item--info">
```

Comment:
- [ ] As mentioned by the previous reviewer I think it would be great if you align the categories to the middle to match the design as much as possible, so please As required for the project your app to follow the the [the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) from Behance, kindliy ensure that you update your current design by using the typography(font, size, weight,...), text-transformation, Layout(alignment and position of elements), colors, etc. ✨🚀💯✅

- Your design 
![image](https://github.com/Akorede3133/expense-insight/assets/58553711/1de5b381-aa90-4957-849d-77bc87530143)


- Given
https://user-images.githubusercontent.com/83042742/284735931-fa478296-2a85-4de3-9f58-baa024a17dac.png